apps/events/migrations/0039_remove_event_city_north_east_point_and_more.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.2.3 on 2024-06-14 11:14
+
+from django.conf import settings
+import django.contrib.gis.db.models.fields
+import django.contrib.gis.geos.point
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('events', '0038_alter_event_end_date_alter_event_start_date'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='event',
+            name='city_north_east_point',
+        ),
+        migrations.RemoveField(
+            model_name='event',
+            name='city_south_west_point',
+        ),
+        migrations.RemoveField(
```

Comment:
We can't delete location

Reply:
Location is about specific place like bar or apartment. city is needed separately to store data about city location